client/app/lib/providers/computecontroller.coffee

```diff
@@ -997,6 +999,47 @@ module.exports = class ComputeController extends KDController
           @emit 'StacksInconsistent', stack
 
 
+  sharingStackTemplate: (_id, type) ->
+    remote.api.JStackTemplate.one { _id }, (err, template) ->
+      template.setAccess type
+
+
+  setStackTemplateAccessLevel: (params) ->
```

Comment:
yes, it does. but it's updating stacks individually which needs to be handled by stack itself.